RELEASE.md

```diff
@@ -276,7 +276,10 @@ To publish a stable release:
    1. Update the Mimir version in the following locations:
       - `operations/mimir/images.libsonnet` (`_images.mimir` and `_images.query_tee` fields)
       - `operations/mimir-rules-action/Dockerfile` (`grafana/mimirtool` image tag)
+   1. Update Jsonnet tests: `make build-jsonnet-tests`
+   1. Commit updated tests
 1. Update dashboard screenshots
+   1. Make sure that operations/mimir-mixin-tools/screenshots/.config is configured with a Mimir dev cell/tenant
```

Comment:
can you either point to the file where the structure is listed or just drop this from the instructions (if they run the command they'll get the error message which says what to put in the file)

Reply:
I'll try.

Reply:
I pointed to the relevant script file (with expected file structures), PTAL.